foundation-extensions/mobile-core/configuration/configuration-api-reference.md

```diff
@@ -465,9 +465,43 @@ MobileCore.configureWithFileInAssets("exampleJSONfile.json");
 {% endtab %}
 {% endtabs %}
 
-## clearUpdatedConfiguration (iOS 3.x.x Only)
+## clearUpdatedConfiguration (Android and iOS (AEP 3.x) Only)
```

Comment:
Let's keep the APIs in alphabetical order.
Let's also move the version details under the header, you can use an info block: `This API is only available in Android and iOS (AEP 3.x).`